sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -637,6 +637,32 @@ class QueryExecutionErrorsSuite
       sqlState = "0A000")
   }
 
+  test("FAILED_RENAME_PATH: rename when destination path already exists") {
+    withTempPath { p =>
+      withSQLConf(
+        "spark.sql.streaming.checkpointFileManagerClass" -> classOf[FileSystemBasedCheckpointFileManager].getName,
```

Comment:
https://github.com/apache/spark/blob/9af216d7ac26f0ec916833c2e80a01aef8933529/sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala#L451-L454

hmm..  because the scalastyle check is off in line 451 and only the throwerror related check is on in line 454, the line 643 and line 651 are more than 100 characters long but not checked

Reply:
@MaxGekk  make a pr restore the code style check https://github.com/apache/spark/pull/38812

Reply:
@ibuder I have merged the PR https://github.com/apache/spark/pull/38812. Could you rebase on the recent master and fix a coding style issue.

Reply:
Done. please re-review